modules/admin_manual/partials/maintenance/major_release_note.adoc

```diff
@@ -16,42 +16,42 @@ Here are some examples:
 |===
 |Version
 |Can Upgrade to {latest-server-download-version} ?
-|Requirements
+|Requirements (always check the xref:{latest-server-version}@server:installation/system_requirements.adoc[System Requirements] too)
 
 |10.X.Y
 |Yes
-|
+| PHP 7.2 to 7.4
```

Comment:
This table needs expanding. Various 10.X supported different subsets of 7.0 7.1 7.2 7.3 7.4 over the recent years.
Somebody should look through all the release notes and make a full table. For example, maybe 10.0 does not support PHP 7.4. And 10.9 only supports PHP 7.3 and 7.4
@mmattel